Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "81":
+    id: "81"
+    taskid: 9793d56b-9b39-491c-8c84-0281dd229584
+    type: playbook
+    task:
+      id: 9793d56b-9b39-491c-8c84-0281dd229584
+      version: -1
+      name: Crowdstrike Falcon - Isolate Endpoint
+      description: This playbook will auto isolate endpoints by the device ID that was provided in the playbook.
+      playbookName: Crowdstrike Falcon - Isolate Endpoint
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      Device_id:
+        complex:
+          root: incident
+          accessor: agentsid
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 970,
+          "y": 2600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "82":
+    id: "82"
+    taskid: 99231bed-c5f1-4160-8a69-d05d21436797
+    type: regular
+    task:
+      id: 99231bed-c5f1-4160-8a69-d05d21436797
+      version: -1
+      name: Close Alert - False Positive
+      description: Closes the alert in Cortex XSIAM
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      closeNotes:
+        simple: Alert Resolved as False Positive.
+      closeReason:
+        simple: False Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1720,
+          "y": 2990
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "89":
+    id: "89"
+    taskid: 0e60cbd9-4688-42a6-85c7-b8c4f13acda2
+    type: title
+    task:
+      id: 0e60cbd9-4688-42a6-85c7-b8c4f13acda2
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "107"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 145
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "91":
+    id: "91"
+    taskid: 85443065-cdf3-4d67-81ba-0a218af503c2
+    type: condition
+    task:
+      id: 85443065-cdf3-4d67-81ba-0a218af503c2
+      version: -1
+      name: No suspicious artifacts were found - How would you like to proceed?
```

Comment:
in the name of the task remove the question. "- How would you like to proceed?"

Reply:
Done